code/HabitTracker/app/src/main/java/com/example/habittracker/activities/HomeActivity.java

```diff
@@ -71,5 +65,61 @@ public void onClick(View view) {
                 startActivity(intent);
             }
         });
+
+        // TODO: Remember to change this with getuser when you are not testing anymore
+        // snapshot
+        DatabaseManager
+                .get()
+                .getHabitsColRef(SharedInfo.getInstance().getCurrentUser().getUsername())
+                .addSnapshotListener(
+                new EventListener<QuerySnapshot>() {
```

Comment:
Something to be done in the future. You should use or modify (as needed) the getAllHabits function in the DatabaseManager class. This logic should not be taking place in the Controller class.  TAs will not like this.

Reply:
Yes, this was written 2 weeks ago ish. Before we decided to do anything like that. I don't even know what getAllHabits does. 

I would like to substitute it with my function but other people are using it (only John) so I didn't touch it for now.